src/Splitio-net-core/Services/Cache/Lru/CacheEnumerator.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Mark Davis. All rights reserved.
```

Comment:
I think that you are missing here: license and repo url

Reply:
You can follow this as an example https://github.com/splitio/javascript-client/blob/development/src/utils/lrucache/linkedlist.js#L2